### DISCUSSION TITLE
How can I show these items in a pdf output?

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForce#51] [How to] [answered]
Hi, 

Hoping someone can help me with this pdf creation issue.

I am currently using Version 7.0.2 I currently have this module:
![FireShot Capture 003 - Scaffolding Inspection Frank Robbins - sgs monster-it co uk](https://github.com/user-attachments/assets/f47bb9db-d00c-4d90-b703-d6015484d8d7)

I want to include the tick boxes so if they have a Tick it will come up with a green tick or if unticked a red X,
then on the pdf output  Where i've got Scaffold Unsafe with the color and icon i want that output on the pdf. Currently it's only outputting the text.

Here is my current PDF output code wise.
`<div class="container">
<table align="center" border="1" cellpadding="1" cellspacing="1" style="width:100%">
	<tbody>
		<tr>
			<td><strong>Inspector Name:</strong>&nbsp;$(record : inspectiondate)$<br />
			<strong>Inspection Date:</strong>&nbsp;$(record : inspectiondatefield)$<br />
			<strong>Client:</strong>&nbsp;$(record : clientjobsummary)$<br />
			<strong>Customer:</strong>&nbsp;$(record : customersummary)$</td>
			<td>&nbsp;</td>
			<td><img align="right" alt="Logo" border="0" class="Logo" hspace="0" src="logo.png" style="border:0px solid #000000; height:120px; margin:0px; width:115px" vspace="0" /></td>
		</tr>
	</tbody>
</table>
</div>

<div>&nbsp;</div>

<h2><span style="color:#000000"><span style="background-color:#dddddd">Job Summary:</span></span></h2>

<p><span style="color:#000000"><span style="background-color:#dddddd">$(record : jobsummary)$<br />
<br />
<strong>Inspectors Comments:&nbsp;</strong>$(record : inspeccomments)$<br />
<br />
<strong>Additional Comments:&nbsp;</strong>$(record : othercommentsbasic)$</span></span><br />
&nbsp;</p>

<h6>Document is intended for use by St George Scaffolding Southwest Ltd.<br />
Company Registrered England &amp; Wales: 11929040.</h6>

<div style="page-break-after: always"><span style="display:none">&nbsp;</span></div>

<h2>Footings</h2>

<table border="1" cellpadding="1" cellspacing="1" style="width:100%">
	<tbody>
		<tr>
			<td style="text-align:center">1. Is the ground suitable for scaffolding?</td>
			<td>$(record : suitablefootingscheck)$</td>
			<td style="text-align:center">Comments about the ground not being suitable</td>
			<td>$(record : groundtextarea)$</td>
		</tr>
	</tbody>
</table>
<strong>Is scaffolding safe:</strong> $(record : inpecdesig)$`

**Comment by rskrzypczak** [comment, marked as answer]
try to use twig
simple example:
```html
  {% if YFParser('$(record : private)$ == 1') %} 
     <span style="color:#008000"> OK </span> 
  {% else %} 
     <span style="color:#ff0000"> X </span> 
  {% endif %}
```

**Reply by MonsterITServices** [reply]
Okay, but how do i create a twig. I'm okay at creating modules, simple php files etc but never worked with composer.

**Reply by rskrzypczak** [reply]
you don't need to upload anything. The twig parser is implemented by default in PDF templates. Just use it.

**Reply by rskrzypczak** [reply]
Just copy my example and change the name of the `private` field to the one you need

**Reply by MonsterITServices** [reply]
Done that but if unticked it does not show any output, just blank and if ticked will show yes, but the code does not seem to work. I think i've put it in the wrong place maybe.

i take it i need to put this in my pdf template?

 here is my test code on the fields.
![Screenshot 2024-11-05 105343](https://github.com/user-attachments/assets/475532ac-11ad-42db-b7d5-b82102212321)

**Reply by rskrzypczak** [reply]
> i take it i need to put this in my pdf template?

yes

try to put your code in `source code`

**Reply by MonsterITServices** [reply]
Thank you, That works 100% :)
